File: Unidad_1/Cancelacion_resta.java
package cancelacion_resta;
public class Cancelacion_resta {
    public static void main(String[] args) {
        double x = 1234567890.1234561;
        double y = 1234567890.1234560;
        double resultado = x - y;
        System.out.println("Resultado esperado: "+0.0000001);
        System.out.println("Resultado real: " + resultado+" Erp"); 
    }
}

File: Unidad_1/3-Codigos/Cancelacion_resta.py
def main():
    x = 1234567890.1234561
    y = 1234567890.1234560
    resultado = x - y
    
    print(f"Resultado esperado: {0.0000001}")
    print(f"Resultado real: {resultado} Erp")

if __name__ == "__main__":
    main()